data/remnant missions.txt

```diff
@@ -495,3 +560,772 @@ mission "Remnant: Technology Available"
+	on offer
+		conversation
+			`As you step out into the icy chill of Viminal, you get a message from Tali, the engineering prefect, asking if you could meet her in the cafeteria.`
+			choice
+				`	(Not right now.)`
+					defer
+				`	(Accept her invitation.)`
+			`	Replying that you can, you make your way through the crowds and find an empty table. Moments later Tali appears and grabs two trays from the dispenser before heading towards you. She settles down in the opposite chair with a look of exhaustion.`
+			`	"It is good to see you again, <first>," she sings softly. When you sign a reply she sits up and smiles. "My, you are a quick learner," she signs. "Great, this is much faster."`
+			`	"I haven't fully..." (She makes a sign that you don't understand, but seems similar to the signs for "break" and "understand".) "...the technology you retrieved for us, but we should be ready to produce them soon. In the meantime, though, they are showing us that humanity has made significant progress since we departed. Could you help us catch up with what has happened out there?"`
+			choice
+				`	"Yes. What can I do to help?"`
+				`	"Sorry, not interested."`
+					decline
+			branch library
+				has "Human Cultural Archives: done"
+			`	"Excellent! According to a friend of mine who is versed in history from the war, there were rumors that a grand library was being built in the Deep to house all of human knowledge. If it exists, that would likely be where to go."`
+				accept
+			label library
+			`	"Excellent! According to a friend of mine who is versed in history from the war, there were rumors that a grand library was being built in the Deep to house all of human knowledge. If it exists..." she stops when she notices you nodding with a knowing expression. "You have already been there? Well, you certainly get around, captain <last>. I suppose that makes things rather straight forward."`
+				accept
+	on stopover
+		payment -40
+		conversation
+			`After having traveled clear across the galaxy, you finally have arrived at the archives of human history. On the direction of the librarians you locate the gift shop and buy a "souvenir copy" of the archive. You idly wonder if you should buy a couple spare copies just in case, but decide against it.`
+	
+	on complete
+		conversation
+			`Having acquired a copy of the archive, you gently set <ship> down on Viminal hoping that this information is helpful. The port seems unusually busy, but the comm channels are quiet beyond the bare minimum to keep traffic organized.`
+			`	Before you have even taken the time to shutdown the engines Tali is halfway up the side of your ship securing a refueling hose and charging cable. As you reach for the comlink she scrambles up to the bridge windows and quickly rattles off a series of signs at you. All you can make out is "Keep it hot, incoming!" Before you can frame a reply she has raced across your wing and leapt to the Albatross parked next to you.`
```

Comment:
"Shutdown" is only concatenated as a noun; it should be split into "shut down" as a verb (as in, "shutting the engines down" or "shutting down the engines", not "shutdowning the engines").

Reply:
changed.